docs/executor-artillery.md

```diff
@@ -0,0 +1,131 @@
+# Artillery.io performance tests
+
+
+## Test Environment
+
+Let's assume that our SUT (Service Under Test) is an internal Kubernetes service which has
+ClusterIP `Service` created and is exposed on port `8088`. The service name is `testkube-api-server`
+and is exposing the `/health` endpoint that we want to test.
+
+To call the SUT inside a cluster:
+
+```sh
+curl http://testkube-api-server:8088/health
+```
+
+Output:
+
+```sh
+200 OK
+```
+
+## Create a test manifest
+
+The Artillery tests are defined in declarative manner, as YAML files.  
+Let's save our test into `test.yaml` file with content below:   
+
+```yaml
+config:
+  target: "http://testkube-api-server:8088/health"
+  phases:
+    - duration: 60
+      arrivalRate: 5
+      name: Warm up
+    - duration: 120
+      arrivalRate: 5
+      rampTo: 50
+      name: Ramp up load
+    - duration: 600
+      arrivalRate: 50
+      name: Sustained load
+```
+
+Now we can create a new Testkube based on the saved Artillery test deifnition.
```

Comment:
new Testkube test

Reply:
definition

Reply:
We can use Add a suggestion option to be more efficient 😄